src/core_ocean/shared/mpas_ocn_time_varying_forcing.F

```diff
@@ -0,0 +1,461 @@
+    block => domain % blocklist
+    do while (associated(block))
+
+       ! apply drag law to compute wind stress from wind speed
+       if (trim(config_atmospheric_forcing_type) == "WINDPRES") then
+          call post_atmospheric_forcing(block)
+       endif
+
+       block => block % next
+    end do
+
+  end subroutine atmospheric_forcing!}}}
+
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  post_atmospheric_forcing
+!
+!> \brief Calculate wind stress from wind speed and apply ramp
+!> \author Steven Brus, LANL
+!> \date 1/8/2019
+!> \details
+!>  Uses drag law from Garratt, J.R., 1977. Review of drag coefficients
+!>  over oceans and continents. Monthly weather review, 105(7), pp.915-929.
+!>
+!
+!-----------------------------------------------------------------------
+
+  subroutine post_atmospheric_forcing(block)!{{{
+
+    type (block_type), pointer :: block
+
+    type (MPAS_time_type) :: currentForcingTime
+
+    type (mpas_pool_type), pointer :: &
+         mesh, &
+         forcingPool, &
+         timeVaryingForcingPool, &
+         diagnosticsPool
+
+    real(kind=RKIND), dimension(:), pointer :: &
+         windSpeedU, &
+         windSpeedV, &
+         windSpeedMagnitude, &
+         windStressZonal, &
+         windStressMeridional, &
+         atmosPressure, &
+         atmosphericPressure
+
+    real(kind=RKIND) :: &
+         windStressCoefficient, &
+         rhoAir, &
+         ramp, &
+         currentTime
+
+    real(kind=RKIND), pointer:: &
+        config_forcing_ramp, &
+        daysSinceStartOfSim
+
+    integer, pointer :: &
+         nCells
+
+    integer :: &
+         iCell
+
+    call MPAS_pool_get_config(block % configs, "config_time_varying_atmospheric_forcing_ramp", config_forcing_ramp)
+
+    call MPAS_pool_get_subpool(block % structs, "mesh", mesh)
+    call MPAS_pool_get_subpool(block % structs, "forcing", forcingPool)
+    call MPAS_pool_get_subpool(block % structs, "timeVaryingForcing", timeVaryingForcingPool)
+    call MPAS_pool_get_subpool(block % structs, "diagnostics", diagnosticsPool)
+
+    call MPAS_pool_get_dimension(mesh, "nCells", nCells)
+
+    call MPAS_pool_get_array(diagnosticsPool, "daysSinceStartOfSim", daysSinceStartOfSim)
+    call MPAS_pool_get_array(timeVaryingForcingPool, "windSpeedU", windSpeedU)
+    call MPAS_pool_get_array(timeVaryingForcingPool, "windSpeedV", windSpeedV)
+    call MPAS_pool_get_array(timeVaryingForcingPool, "windSpeedMagnitude", windSpeedMagnitude)
+    call MPAS_pool_get_array(forcingPool, "windStressZonal", windStressZonal)
+    call MPAS_pool_get_array(forcingPool, "windStressMeridional", windStressMeridional)
+
+    call MPAS_pool_get_array(timeVaryingForcingPool, "atmosPressure", atmosPressure)
+    call MPAS_pool_get_array(forcingPool, "atmosphericPressure", atmosphericPressure)
+
+    rhoAir = 1.225_RKIND
+
+    ramp = tanh((2.0_RKIND*daysSinceStartOfSim)/config_forcing_ramp)
+
+    do iCell = 1, nCells
```

Comment:
The model is running throughout the timestep with multiple threads.  All loops need to be bounded by:
```
+    !$omp do schedule(runtime) private(iCell)
     do iCell = 1, nCells
          ...
     enddo
+    !$omp end do
```
fixing that now.